metals/src/main/scala/scala/meta/internal/metals/PopupChoiceReset.scala

```diff
@@ -28,7 +28,6 @@ class PopupChoiceReset(
     val result = if (value == BuildTool) {
       scribe.info("Resetting build tool selection.")
       tables.buildTool.reset()
-      tables.buildServers.reset()
```

Comment:
Because we only change the build tool? So either we trust the user to be reasonable and not chose `mill` build tool for `sbt` build server. I they do it is still mostly fine. If we reset buildServer choice we'd have to do full reconnect (`quickConnect` & `slowConnect`) to be connected to correct build server for metals state consistency.

Reply:
Let's say that we have to servers in `.bsp` without resetting we will have to manually do the switch to the other tool, no?

Reply:
No, since those "forceBuildServer", so it will switch automatically in `slowConnect`.